src/mobx.ts

```diff
@@ -39,8 +39,9 @@ try {
     function testCodeMinification() {}
     if (
         testCodeMinification.name !== "testCodeMinification" &&
-        process.env.NODE_ENV !== "production" &&
```

Comment:
Are you sure this doesn't break the minification with bundlers that inject these constants into the code, such as webpack?